controller/users.go

```diff
@@ -117,6 +147,7 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
+		keycloakUserProfile.Update(tokenString, "http://sso.prod-preview.openshift.io/auth/realms/fabric8-test/account")
```

Comment:
The service impl needs to be passed into the controller. That would allow us to send in a dummy service for test and not rely on KC.